File: timestamp/timestamp.php
---
permalink: /cgi/timestamp/obsolete.php
---

<?php
session_start();
session_set_cookie_params(3600);

date_default_timezone_set("{{ site.timezone }}");

$timestamp = date('Y.m.d-h:i:s', time());

$milliseconds = floor(microtime(true) * 1000);

#echo($timestamp . "_" . time() . "_" . $milliseconds);

header('Content-Type: text/plain; charset=utf-8');
echo($timestamp . '.' . microtime(true));
?>

File: timestamp/timestamp.lua
---
permalink: /cgi/timestamp/index.lua
---

-- Set the timezone (not directly supported in Lua)
-- You'll need to handle timezone conversion manually if needed

-- Get current time
local timestamp = os.date("%Y.%m.%d-%H:%M:%S")

-- Get current time in milliseconds
local milliseconds = os.time() * 1000

-- Output the HTTP header
io.write("Content-Type: text/plain; charset=utf-8\r\n\r\n")

-- Output result
io.write(timestamp .. '.' .. milliseconds)

